openshift/templates/issuer-agent/issuer-agent-deploy.lsbc.param

```diff
@@ -21,7 +22,7 @@ AGENT_ADMIN_PORT=8024
 ADMIN_API_KEY=[a-zA-Z0-9]{16}
 AGENT_HTTP_PORT=8021
 GENESIS_FILE_URL=https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis
-LOG_LEVEL=WARNING
+LOG_LEVEL=DEBUG
```

Comment:
This can be updated as needed when actually debugging, I would leave it set at `WARNING` for deployments.